server/src/main/kotlin/org/javacs/kt/Configuration.kt

```diff
@@ -35,10 +35,16 @@ public data class ExternalSourcesConfiguration(
     var autoConvertToKotlin: Boolean = false
 )
 
+public data class GradleConfiguration(
+    /** The JDK to use to launch gradle, the default is JAVA_HOME. */
+    val home: StringBuilder = StringBuilder("default")
```

Comment:
I would just go with `String`. This isn't performance-critical, `StringBuilder` isn't really designed to be stored and introducing shared mutability here might introduce a source of confusion.